services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/extensions/dynamodb/mappingclient/functionaltests/AsyncBatchWriteItemOperationTest.java

```diff
@@ -187,7 +187,10 @@ public void singlePut() {
         List<WriteTransaction> writeTransactions =
             singletonList(WriteTransaction.create(mappedTable1, PutItem.create(RECORDS_1.get(0))));
 
-        enhancedAsyncClient.execute(TransactWriteItems.create(writeTransactions)).join();
```

Comment:
Did we get rid of create? I think we agreed that every enhanced request should have a create(...) method that made the most sense for the most likely/simply use-cases. Eg: PutItemEnhancedRequest.create(myItem);

Reply:
I think that one went out the window for 3 of the 4 operations. I will put it back in.